skyportal/plot.py

```diff
@@ -600,6 +600,15 @@ def photometry_plot(obj_id, user, width=600, height=300):
     plot.yaxis.axis_label = 'AB mag'
     plot.toolbar.logo = None
 
+    obj = DBSession().query(Obj).get(obj_id)
+    if obj.dm is not None:
+        plot.extra_y_ranges = {
+            "Absolute Mag": Range1d(start=ymax - obj.dm, end=ymin - obj.dm)
+        }
+        plot.add_layout(
+            LinearAxis(y_range_name="Absolute Mag", axis_label="m - μ"), 'right'
```

Comment:
This labeling is really confusing since you have to know how to orient your head to understand that the axis is saying "m - μ" (otherwise it looks like garbage). Maybe say `M_<sub>absolute</sub> (no K-correction)`?

Reply:
I changed it to m - DM. I considered `M_abs (no K-correction)` but this leaves out the fact that there is also no EBV correction. Adding the "no ebv correction" makes the label too long. I think this is more readable than m - µ was, and is also sufficiently precise to avoid any confusion about what corrections (or lack thereof) have been made to the photometry

Reply:
<img width="565" alt="Screen Shot 2020-09-26 at 12 09 34 PM" src="https://user-images.githubusercontent.com/2769632/94348436-2cada480-fff1-11ea-81e9-99feddc88db7.png">